invenio_logging/structlog.py

```diff
@@ -0,0 +1,69 @@
+import structlog
+import logging
+from logging import StreamHandler
+import sys
+
+class LoggerFactory:
```

Comment:
Not clear how all this is used, it would be nice to see an example.
It feels like that the class is not needed, but a simple func is enough.

Reply:
Right! Once I push to the vocabularies PR and it's ready to review we will see it there, but basically we can use it by simply doing:

```python
stream_logger = LoggerFactory.get_logger("datastreams-" + stream)
stream_logger.info("Starting processing")
stream_logger.error(...)
stream_logger.exception(...)
```

Yes, we could change it to be simply a func that returns the logger if it's better.